exercises/patterns/printPlus/printPlus.js

```diff
@@ -38,6 +38,15 @@ function printPlus(height) {
   }
 
   for (let i = 0; i < height; i++) {
+    if (i === (height - 1) / 2) {
+      let numChars = height;
+      helpers.printCountTimes('#', numChars);
```

Comment:
Nice solution! You can remove line 42 and just pass `height` as an argument to `printCountTimes` as in `helpers.printCountTimes('#', height);` to simplify your code.